File: Python/card.py
from payment import Payment

class Card(Payment):
    number      = int
    cvv         = int
    date        = int
    
    def __init__(self, id, number, cvv, date):
        super().__init__(id)
        self.cvv    = cvv
        self.number = number
        self.date   = date

File: PHP/Card.php
<?php
    require_once('payment.php');
    class Card extends Payment {
        public $number;
        public $cvv;
        public $date;

        public function __construct($id, $number, $cvv, $date){
            parent::__construct($id);
            $this->number = $number;
            $this->cvv = $cvv;
            $this->date = $date;
        }

        public function printDataCard(){
            echo "Number: $this->number , CVV:  $this->cvv , Date: $this->date";
        }
    }
?>
